src/semgrep_agent/constants.py

```diff
@@ -1,3 +1,5 @@
 SUPPORT_EMAIL = "support@r2c.dev"
 
 PRIVACY_SENSITIVE_FIELDS = {"syntactic_context"}
+
+PUBLISH_TOKEN_LENGTH = 16
```

Comment:
After generating a test token in the cloud app UI, it looks like the length is 64, not 16. Just wanted to double check.

Reply:
ah whoops

Reply:
```postgres=> select distinct char_length(token) from token;
 char_length
-------------
          60
(1 row)```